### PR TITLE
Guard against iOS 9.0+ API usage

### PR DESCRIPTION
**Pull request by EricMuller22**
Open to other ways of protecting against this... spending my Monday trying to get to a spot where our build has 0 warnings.

[Context](https://twitter.com/TheProfessor22/status/980874581941432320)

### CODE REVIEW

File: Aardvark/ARKEmailBugReporter.m [hunk 1]
```diff
@@ -121,10 +121,12 @@ @implementation UIViewController (HierarchyDescription)
 
 - (void)_ARK_appendRecursiveViewControllerMappingToMapTable:(NSMapTable<UIView *, UIViewController *> *)mapTable;
 {
-    [mapTable setObject:self forKey:self.viewIfLoaded];
-    
-    for (UIViewController *childViewController in self.childViewControllers) {
-        [childViewController _ARK_appendRecursiveViewControllerMappingToMapTable:mapTable];
+    if (@available(iOS 9.0, *)) {
```
Comment (dfed) [inline Aardvark/ARKEmailBugReporter.m]: Should we make including the view controller hierarchy a feature that's only available in iOS 9+?

**Comment by NickEntin** [comment]
Oops, looks like it’s `viewIfLoaded` that’s iOS 9+.  Instead of an `@available` (which makes this a breaking change), we should just use `isViewLoaded` and `view`.

**Comment by EricMuller22** [comment]
(i.e. in an `else` @NickEntin?)

**Comment by NickEntin** [comment]
I think you can just replace:
```
[mapTable setObject:self forKey:self.viewIfLoaded];
```
with
```
if (self.viewIsLoaded) {
    [mapTable setObject:self forKey:self.view];
}
```

**Comment by EricMuller22** [comment]
Gorm on my end, up to @dfed if he wants to keep the change to compiler settings or not.

**Comment by dfed** [comment]
LGTM!

**Pull request merged**